packages/dds/tree/src/util/brand.ts

```diff
@@ -117,3 +117,30 @@ export function brand<T>(
 ): T {
 	return value as T;
 }
+
+/**
+ * Adds a type {@link Brand} to a value, while preserving the exact type of the value being branded.
```

Comment:
We should put a link to this in the docs on the non-constant `brand` function that says something along the lines of "If you're branding a value that you want to be constantly-typed, use {@link brandContant}" for discoverability.

Reply:
Done.